managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedger.java

```diff
@@ -138,6 +138,20 @@
      */
     public ManagedCursor openCursor(String name) throws InterruptedException, ManagedLedgerException;
 
+    /**
+     * Open a ManagedCursor in this ManagedLedger.
+     * <p>
+     * If the cursors doesn't exist, a new one will be created and its position will be at the end of the ManagedLedger.
+     *
+     * @param name
+     *            the name associated with the ManagedCursor
+     * @param initializeOnLatest
+     *            the flag tell the method wthether it should intialize the cursor at latest position or not.
+     * @return the ManagedCursor
+     * @throws ManagedLedgerException
+     */
+    public ManagedCursor openCursor(String name, boolean initializeOnLatest) throws InterruptedException, ManagedLedgerException;
```

Comment:
rather than using a boolean I'd prefer having an enum here. Reading code with: 

```java
managedLedger.openCursor("name", true);
```

doesn't convey what the `true` means.

```java
managedLedger.openCursor("name", CursorPosition.InitializeOnLatest);
managedLedger.openCursor("name", CursorPosition.InitializeOnEarliest);
```

Reply:
Another option here is to reuse  `Position` class: 

eg: 
```java
managedLedger.openCursor("name", Position.earliest);
```

Reply:
Thanks for spending on reviewing works, I'll modify the code and apply the enum implementation. 👍